src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static pushAnimal(animal) {
+    Animal.alive.push(animal);
+  }
```

Comment:
I think that you don't need this

Reply:
Agree, looks like a redundant method. Let's get rid of it.